task-1/task1.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />

```

Comment:
This meta tag is needed to configure document mode for IE browser.
We had an agreement that our work should be carried out taking into account the support of the latest versions of modern browsers. Modern browsers simply ignore this line of code.
Just don't use it in the future.

Reply:
vs code inserts it by default, I'll remove it